spec/models/vim_performance_state_spec.rb

```diff
@@ -1,5 +1,40 @@
 RSpec.describe VimPerformanceState do
-  describe ".capture_host_sockets" do
+  let(:capture_time) { 5.hours.ago.beginning_of_hour }
+
+  describe ".capture" do
+    it "uses now for Infra" do
+      hardware = FactoryBot.create(:hardware, :cpu_sockets => 2)
+      host = FactoryBot.create(:host, :hardware => hardware)
+      state = VimPerformanceState.capture(host, capture_time)
+
+      expect(state.timestamp).to be_within(1.minute).of(Time.now.beginning_of_hour)
```

Comment:
This is a very weird condition (I fully understand that this is the way it works currently) and a future reader will not understand this.  I think it would be helpful to have a `NOTE` comment or something similar describing _why_ now is used for Infra and thus capture_time is ignored.  Not sure if that note should live here or in the code or both.

Reply:
luckily we no longer need to deal with the nuance of infra vs containers